releasenotes/notes/windowssites-0b52e956621bef1d.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Added new command line properties to the Windows installer which allow for setting site specific configuration.
```

Comment:
let's update the release note here (should just mention "the `dd_url` option of the different agent components or something similar, without mentioning "site")

Reply:
I don't understand.  I took out the description of all the options b/c your previous comment said that was too specfic for the relnote; that the changes to the doc page was fine.  I can put those descriptions (except for `site`) back in...

Reply:
Sorry for the confusion, all I meant was that I felt the release note should not mention `site` at all. No need to change it now though